src/components/base-form.njk

```diff
@@ -53,10 +53,8 @@
 
 {% block bodyEnd %}
   {% block scripts %}
-    <input type="hidden" aria-hidden=true value="{{gaTaxonomyLevel2}}" id="gaTaxonomyLevel2">
-    <script type="text/javascript" src="/public/javascripts/analytics.js"></script>
     <script type="text/javascript" src="/public/javascripts/all.js"></script>
-    <script type="text/javascript" src="/public/javascripts/application.js"></script>
```

Comment:
Is application.js intentionally removed here?

Reply:
yes. CRI IPV frontend used to merge all the logic around analytics and cookie banner into 1 file: application.js.
But now this logic is inside Analytics package.